### ISSUE TITLE
4.6.14 Settings issue

### ISSUE THREAD

**Issue by jrdesy** [eirannejad/pyRevit#508]
Revit version: 2018.3.2 - 2017.2.3 -  2016
Windows 7

Hello Ehsan
I was using version 4.6.5 for the past few months. This morning I upated to version 4.6.14.
I first noticed a problem when my custom extension failed to appear.
Selecting 'just save settings' causes revit to crash.
Selecting 'save settings and reload' gives me this error message.

![pyrevit error1](https://user-images.githubusercontent.com/47362411/52291649-d0d86880-2940-11e9-9e51-1ce43bd6c334.jpg)
![pyrevit error2](https://user-images.githubusercontent.com/47362411/52291667-e0f04800-2940-11e9-92c0-aa5d28d71bb4.jpg)

I tried reverting back to 4.6.5 but now i get this error message when accessing settings.

![pyrevit error 3](https://user-images.githubusercontent.com/47362411/52292183-1fd2cd80-2942-11e9-8307-8206f085e32b.jpg)

I then tried repairing with the 4.6.14 exe but I end up with the first issue.
I now only have version 4.6.14 installed.

I tried multiple clean reinstalls, manual uninstalls and reboots but nothing seem to work

Other thoughts
I recently installed the RVTTools addin
Tried a 'dummy' extension
Same error message for all Revit versions, currently using 2018.3.2

Thanks for your time and help, I will do my best to help you figure out the problem.

**Comment by eirannejad** [comment]
The configuration file format has changed slightly. It should have updated your config file automatically but there might have been an issue. Delete your config file under `%appdata%/pyRevit` and restart Revit. Add the extensions paths again,

**Comment by jrdesy** [comment]
Closed Revit then deleted file 'pyRevit_config.ini' under AppData\Roaming\pyRevit
I now get a arror message when selecting settings followed by the same error when trying to add a file path.


![settingserror1](https://user-images.githubusercontent.com/47362411/52296524-9bd21300-294c-11e9-8a72-6cea3cd14bcb.jpg)
.

**Comment by eirannejad** [comment]
Woops. This one is my bad. Fixed and will be pushed with 4.6.15
Download [userconfig.zip](https://github.com/eirannejad/pyRevit/files/2833479/userconfig.zip) and replace the `pyrevitlib/pyrevit/userconfig.py` to patch.

**Comment by jrdesy** [comment]
I replaced the userconfig.py file under pyrevitlib/pyrevit.
The Ironpython Traceback error is gone but not the error message when accessing settings.

![settingserror](https://user-images.githubusercontent.com/47362411/52299856-7b0dbb80-2954-11e9-899d-f777b8c2278a.jpg)

I understand the fix for this will be pushed with 4.6.15

Unfortunately my extension stills fails to appear.
Tried deleting file 'pyRevit_config.ini' and reentering the path.

Should I close this issue and start another one?

**Comment by Hazen12345** [comment]
Ehsan Rebooted and that looks like it:

Just added pyrevit to another PC, and the extensions icon is on it, could I have moved it or deleted it on my PC?

Thanks;

James

**Comment by eirannejad** [comment]
Maybe. Did u delete the config file?

**Comment by jrdesy** [comment]
Ehsan, looking at the deployment video right now, 
Haven't been able to keep up with the new features and my programming skills are intermediate.

Will comment on progress.

Thank you for your help.

**Comment by Hazen12345** [comment]
Ehsan, hit the search button and typed Extensions up popped the Extension Manager!

On another note the python tutorials is there a blog for us rookies, as I found an issue with the rps_init linked on the youtube tutorisl.
This code was missing so I had to retype from the one you used on the video.
I still don’t see the wall information, when you do the python snippet part cl. El (cl) print (el)
Or something like that, have to watch the video again!


[cid:image001.png@01D4BD6A.08F84B60]
